### PR TITLE
git diff: --cached → --staged

### PR DESCRIPTION
**Pull request by hginzel**
To be consistent with term “staging area”.

### CODE REVIEW

**Comment by ben** [comment]
I like the change from `--cached` to `--staged`, but it's the prose changes that are causing more thought. Is this everywhere we mention the index? I think it's valuable to mention that "staging area" and "index" mean the same thing, since a user will run into both of those in the man pages. But being consistent is good, too.

File: book/02-git-basics/sections/recording-changes.asc [hunk 1]
```diff
@@ -506,7 +506,7 @@ Changes to be committed:
 ----
 
 The next time you commit, the file will be gone and no longer tracked.
-If you modified the file and added it to the index already, you must force the removal with the `-f` option.
+If you modified the file and added it to the staging area already, you must force the removal with the `-f` option.
```
Comment (YueLinHo) [inline book/02-git-basics/sections/recording-changes.asc]: perhaps using:
```
[...] added it to the index(staging area) already, [...]
```

**Comment by ben** [comment]
@ginzel, are you still interested in getting this merged?

**Comment by aollier** [comment]
The `SYNOPSIS` part of the [documentation](https://git-scm.com/docs/git-diff "git diff documentation") showcases the `--cached` option. We can read in the `DESCRIPTION` part that:
> --staged is a synonym of --cached.

So from this point of view, I think we should let the `--cached` option. Then the reader won't be confused when reading the documentation.

**Comment by aizulaiz** [comment]
update

**Comment by aollier** [comment]
@ben I think this pull request is not a good idea as I explained in my previous comment and @hginzel does not answer.
Do you agree to close it ?

**Comment by ben** [comment]
Agreed. Closing this one out.

**Pull request closed without merging**